src/dbt_osmosis/core/column_level_knowledge.py

```diff
@@ -0,0 +1,39 @@
+import re
```

Comment:
move from src/dbt_osmosis/core/osmosis.py to avoid cross-references with src/dbt_osmosis/core/column_level_knowledge_propagator.py.